api/v1/src/controller/entrepriseController.js

```diff
@@ -39,7 +39,17 @@ export default class EntrepriseController
     }
     deleteEntreprises(id)
     {
-        // Return promise
+        if(!this.eV.checkId(req.body.id))
```

Comment:
Ici il faut supprimer le if else et dans le catch faire un:

e => this.sendJsonResponse(httpStatusCode.internalServerError, e)